src/otx/recipe/semantic_segmentation/segnext_b_tile.yaml

```diff
@@ -0,0 +1,38 @@
+model:
+  class_path: otx.algo.segmentation.segnext.SegNext
+  init_args:
+    label_info: 2
+    model_name: segnext_base
+
+    optimizer:
+      class_path: torch.optim.AdamW
+      init_args:
+        lr: 0.00006
```

Comment:
Is it ok to use very small lr like this?

Reply:
Small learning is inherited from the original recipe. @kprokofi might be able to provide more details.